frontend/src/pages/agreements/list/AgreementsFilterSection.jsx

```diff
@@ -0,0 +1,25 @@
+import "./AgreementsList.scss";
+import AgreementsFilterButton from "./AgreementsFilterButton";
+import AgreementsFilterTags from "./AgreementsFilterTags";
+
+/**
+ * A header section of the agreements page that contains the filters.
+ * @param {Object} props - The component props.
+ * @param {Object} props.filters - The current filters.
+ * @param {Function} props.setFilters - A function to call to set the filters.
+ * @returns {JSX.Element} - The procurement shop select element.
+ */
+export const AgreementsFilterSection = ({ filters, setFilters }) => {
+    return (
+        <div className="padding-top-05 padding-bottom-05 display-flex" style={{ justifyContent: "space-between" }}>
```

Comment:
may be helpful to have this as a custom class in our styles helper file called something like `flex-justify-space-between`